tests/test_conditions.py

```diff
@@ -0,0 +1,48 @@
+from time import monotonic
+
+from pytest import raises
+
+from evol import Population
+from evol.conditions import Condition, MinimumProgress, TimeLimit
+from evol.exceptions import StopEvolution
+
+
+class TestCondition:
+
+    def test_context(self):
+        with Condition(lambda pop: False):
+            assert len(Condition.conditions) == 1
+            with TimeLimit(60):
+                assert len(Condition.conditions) == 2
+        assert len(Condition.conditions) == 0
+
+    def test_check(self, simple_population):
+        with Condition(lambda pop: False):
+            with raises(StopEvolution):
+                Condition.check(simple_population)
+        Condition.check(simple_population)
```

Comment:
missing `assert` ?

Reply:
With the second I "assert does not raise", which doesn't need an explicit assert 😊